unity/Assets/UnitTests/TestPutObject.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using NUnit.Framework;
+using System;
+using UnityEngine;
+using UnityEngine.TestTools;
+using UnityStandardAssets.Characters.FirstPerson;
+
+namespace Tests
+{
+    public class TestPutObject : TestBase
+    {
+        [UnityTest]
+        public IEnumerator TestPutObject_PutNearXY_True()
+        {
+            yield return ExecuteDebugAction("init");
+            yield return ExecuteDebugAction("rr");
+            yield return ExecuteDebugAction("rr");
+            yield return ExecuteDebugAction("ld");
+            yield return ExecuteDebugAction("mr");
```

Comment:
In our group chat about these C# tests we'd talked about not using the debug controller as it effectively creates a "meta" API and will make people nervous to change `DebugInputField.cs`. Is there a way to use the actions by passing a json-formatted string of the python-side act more directly (e.g. similarly as is done when calling `run simple.json` in the input field)? Pinging @ekolve / @mattdeitke in case they know.